packages/@ember/-internals/glimmer/lib/utils/bindings.ts

```diff
@@ -118,6 +118,14 @@ export const AttributeBinding = {
 const DISPLAY_NONE = 'display: none;';
 const SAFE_DISPLAY_NONE = htmlSafe(DISPLAY_NONE);
 
+function emitIsVisibleDeprecation() {
```

Comment:
Unfortunately, I think DRYing this up will not result in the deprecated code path from being completely stripped. I think we should inline the two deprecation calls..